en/pp_limiter_en.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Datenshi Performance Points Limiter"
+old_id: 13
+---
+<div style="text-align:center"><img src="https://user-images.githubusercontent.com/10250068/116959299-c8ae3580-accf-11eb-9ca3-15cd662dbca0.png" /></div>
+
+Hello! this is guide for who got messages from our Bot In-game, we are introduce to you that we have a unique pp limiter, this limiter are mostly prevent abusers/cheater to have more pp, **please note that you are not in banned/restricted mode**. It's just that you won't get more pp, if you play another fresh beatmap you will be get 0pp.
+
+# Limits
+
+These limits are applies both in Relax & Vanilla
+
+- Total PP
+  - 5000pp (STD)
+  - 8888pp (Taiko) 
+  - 8888pp (CTB)
+  - 9999pp (mania)
+- Per Score/Individual PP
+  - Not so disclosed value.
+
+\* PP limit may be changed in future without any notice depending on staff's discretion.
+
+# How to remove the limiter?
+
+1. You need to join our [Discord Server](https://link.troke.id/datenshi) 
+2. Link your datenshi account with your discord account using our [Bot](https://osu.troke.id/discordtokens). If you already linked, you can skip this step.
+3. Go to the channels `#issues-help`, **only this channel**. Post your [screenshot](https://cdn.discordapp.com/attachments/698597147553169429/816492708356423680/screenshot308.jpg) and mention (one of) our **Moderators** or **Community Manager** for whitelist inquiry. Any inquiries done outside the channel will ignored.
+
+## Things to note
+
+- Devs ***__may__*** respond as well, but **please do not ping either of them directly.**
+- Do not contact our staff by DM's. Use the given channel for it. ![ehu](https://user-images.githubusercontent.com/9847780/116959893-1ecbaa80-acc9-11eb-99ce-fd475b89e75c.png)
+
+# Screenshot
+
+![image](https://user-images.githubusercontent.com/10250068/116958891-94864500-acce-11eb-8feb-17b6fbc2a930.png)
+
+# FAQ
+
+- Q: **What if I didn't use Discord?**
+- A: We will not approve any other kind of inquiries for this. Make your Discord account now ![ehu](https://user-images.githubusercontent.com/9847780/116959893-1ecbaa80-acc9-11eb-99ce-fd475b89e75c.png). Otherwise you won't be able to gain more PPs.
```

Comment:
```suggestion
- A: We will not approve any other kind of inquiries other than from Discord, you should make a Discord account ![ehu](https://user-images.githubusercontent.com/9847780/116959893-1ecbaa80-acc9-11eb-99ce-fd475b89e75c.png). Otherwise you won't be whitelisted and be able to gain more PPs.
```
pee poo